hr_timesheet_no_closed_project_task/models/project_task.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+PROJECT_SELECTION = [('template', 'Template'),
+                     ('draft', 'New'),
+                     ('open', 'In Progress'),
+                     ('cancelled', 'Cancelled'),
```

Comment:
@elicoidal it seems this is a fine point of american english versus rest of the world. Anyway, this part was useless anyway, since it's inherited from the related field.